CONTRIBUTING.md

```diff
@@ -211,6 +211,15 @@ To reduce the effort required to review documentation-related changes,
 be viewed by clicking the `details` link on the
 `netlify/tracing-rs/deploy-preview check` on all pull requests.
 
+### Building Documentation
+
+Since we use nightly-only features and lints like `doc(cfg)` and `broken_intra_doc_lints`,
+you should do like this to build docs locally.
```

Comment:
how about something like this?
```suggestion
Tracing's documentation uses nightly-only RustDoc features and lints, like
`doc(cfg)` and `broken_intra_doc_lints`. These features are enabled by 
passing `--cfg docsrs` to RustDoc. Therefore, in order to build Tracing's
documentation the same way it would be built by docs.rs, it's necessary to
use the following command:
```

Reply:
Thank you. I've accepted your suggestion!